docs/flow/concepts/configuration_language.md

```diff
@@ -6,3 +6,86 @@ weight: 300
 ---
 
 # Configuration language
+
+The Grafana Agent Flow _configuration language_ refers to the language used in
+configuration files which define and configure components to run.
+
+The configuration language is called River, a Terraform/HCL-inspired language:
```

Comment:
Link to the RFC?

Reply:
Given that the RFC mainly just talks about justification and not about any language details, I wonder if linking to it would just cause more confusion 🤔

Reply:
I'll leave this as-is for now, but we can consider whether the detailed section for River can link to the RFC somewhere.